modules/articles/client/articles.client.module.js

```diff
@@ -1,7 +1,7 @@
 (function (app) {
   'use strict';
 
-  app.registerModule('articles');
+  app.registerModule('articles', ['core']);
```

Comment:
@mleanos should we add a comment above this line to indicate core is required due to provide deterministic order..forcing core to load first?